packages/pn-pa-webapp/src/redux/NewApiKey/actions.ts

```diff
@@ -1,20 +1,12 @@
-/* eslint-disable sonarjs/no-identical-functions */
-/* eslint-disable @typescript-eslint/no-unused-vars */
-
-
-/*
-  Funzioni mock in attesa del BE
-  Rimuovere tutti i commenti eslint quando pronto.
-*/
 import { createAsyncThunk } from "@reduxjs/toolkit";
+import { ApiKeysApi } from "../../api/apiKeys/ApiKeys.api";
 import { NotificationsApi } from "../../api/notifications/Notifications.api";
 import { UserGroup, GroupStatus } from "../../models/user";
 import { NewApiKeyType } from "./types";
 
 export const saveNewApiKey = createAsyncThunk<string, NewApiKeyType>('saveNewApiKey', async (param: NewApiKeyType, { rejectWithValue }) => {
```

Comment:
I suggest you use `performThunkAction`, cfr. https://pagopa.atlassian.net/wiki/spaces/PN/pages/583173517/Gestione+degli+errori+nelle+chiamate+API . Same for `src/redux/apiKeys/actions.ts`.